tests/integration_tests/build/test_coverage.py

```diff
@@ -23,7 +23,7 @@
 # this contains the frequency while on AMD it does not.
 # Checkout the cpuid crate. In the future other
 # differences may appear.
-COVERAGE_DICT = {"Intel": 84.63, "AMD": 84.60}
+COVERAGE_DICT = {"Intel": 84.43, "AMD": 84.39}
```

Comment:
This is a rather big decrease :(. Are we sure all the code that can be rust tested is indeed tested? I know we will have some awesome python integration tests, but it's nice to add also unit tests (where it is possible).